wpeverest/user-registration
# Enhance - Max upload size and valid file type settings value save and retrieve

### All Submissions:






### Changes proposed in this Pull Request:


This PR tends to support valid file type and max upload size settings in advance settings of the individual file upload field.

### How to test the changes in this Pull Request:

1. Visit form builder and drag a file upload field.
2. See advance settings in field options of the dragged file upload field.
3. Two new settings i.e. valid file type and max upload size will be seen, make use of both settings to see if they work.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Max upload size and valid file type settings value save and retrieve.
